_posts/2020-07-31-status-update.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "31th July 2020: IAC Team Status Update"
+author: david22swan
+categories:
+  - team
+  - status
+tags:
+  - releases
+  - pdk
+  - shoutouts
+  - security
+  - rspec-puppet
+  - java
+  - inifile
+  - docker
```

Comment:
```suggestion
  - docker
  - puppet7
```

Reply:
Oh...and should `rspec-puppet` and `pdk` be in there?

Reply:
...and `java`, `inifile`? Looks like last week's tags

Reply:
Updated tags